modules/swagger-codegen/src/main/resources/swift5/Podspec.mustache

```diff
@@ -0,0 +1,22 @@
+Pod::Spec.new do |s|
+  s.name = '{{projectName}}'{{#projectDescription}}
+  s.summary = '{{projectDescription}}'{{/projectDescription}}
+  s.ios.deployment_target = '9.0'
+  s.osx.deployment_target = '10.11'
+  s.tvos.deployment_target = '9.0'
+  s.version = '{{#podVersion}}{{podVersion}}{{/podVersion}}{{^podVersion}}0.0.1{{/podVersion}}'
+  s.source = {{#podSource}}{{& podSource}}{{/podSource}}{{^podSource}}{ :git => 'git@github.com:swagger-api/swagger-mustache.git', :tag => 'v1.0.0' }{{/podSource}}{{#podAuthors}}
+  s.authors = '{{podAuthors}}'{{/podAuthors}}{{#podSocialMediaURL}}
+  s.social_media_url = '{{podSocialMediaURL}}'{{/podSocialMediaURL}}{{#podDocsetURL}}
+  s.docset_url = '{{podDocsetURL}}'{{/podDocsetURL}}
+  s.license = {{#podLicense}}{{& podLicense}}{{/podLicense}}{{^podLicense}}'Proprietary'{{/podLicense}}{{#podHomepage}}
+  s.homepage = '{{podHomepage}}'{{/podHomepage}}{{#podSummary}}
+  s.summary = '{{podSummary}}'{{/podSummary}}{{#podDescription}}
+  s.description = '{{podDescription}}'{{/podDescription}}{{#podScreenshots}}
+  s.screenshots = {{& podScreenshots}}{{/podScreenshots}}{{#podDocumentationURL}}
+  s.documentation_url = '{{podDocumentationURL}}'{{/podDocumentationURL}}
+  s.source_files = '{{projectName}}/Classes/**/*.swift'{{#usePromiseKit}}
+  s.dependency 'PromiseKit/CorePromise', '~> 4.4.0'{{/usePromiseKit}}{{#useRxSwift}}
+  s.dependency 'RxSwift', '~> 4.0'{{/useRxSwift}}
+  s.dependency 'Alamofire', '~> 4.5.0'
```

Comment:
@plam4u the version of Alomofire included in podspec doesn't compile with Xcode 11 and Swift 5.1.

I needed to upgrade from s.dependency 'Alamofire', '~> 4.5.0' to s.dependency 'Alamofire', '~> 4.9.0'.

Can you also update the podspec please?

Reply:
@plam4u the version of Alomofire included in podspec doesn't compile with Xcode 11 and Swift 5.1.

I needed to upgrade from 4.5.0 to 4.9.0.

Can you update the podspec please?